### ISSUE TITLE
Improve on animations

### ISSUE THREAD

**Issue by bradlegge** [element-hq/element-web#1779]
This has been migrated from the vector design spreadhseet and is still at requirement stage

Blocked on the Vector side (in terms of determining how many more assets are needed)

**Comment by t3chguy** [comment]
Closing in favour of https://github.com/element-hq/element-web/issues/5834
